src/rgw/rgw_rest_s3.cc

```diff
@@ -1013,89 +1008,18 @@ void RGWListBucket_ObjStore_S3v2::send_versioned_response()
 
 void RGWListBucket_ObjStore_S3v2::send_response()
 {
-  
-  if (op_ret < 0)
-    set_req_state_err(s, op_ret);
-  dump_errno(s);
 
-     // Explicitly use chunked transfer encoding so that we can stream the result
-  // to the user without having to wait for the full length of it.
-  end_header(s, this, "application/xml", CHUNKED_TRANSFER_ENCODING);
-  dump_start(s);
-  if (op_ret < 0)
-    return;
-
-   if (list_versions) {
-    send_versioned_response();
-    return;
-  }
 
-
-    s->formatter->open_object_section_in_ns("ListBucketResult", XMLNS_AWS_S3);
-  if (!s->bucket_tenant.empty())
-    s->formatter->dump_string("Tenant", s->bucket_tenant);
-  s->formatter->dump_string("Name", s->bucket_name);
-  s->formatter->dump_string("Prefix", prefix);
```

Comment:
Have you missed invoking send_common_response() here?

Reply:
Done